src/test/functional/udf.spec.ts

```diff
@@ -1,154 +1,167 @@
 import * as assert from "assert";
-import * as Stream from "stream";
-import {
```

Comment:
Looks like the lint step doesn't check for unused imports? I think we should add that in a different PR.

Reply:
Agreed. tslint doesn't check for that with the current settings. it should